odoo/http.py

```diff
@@ -331,14 +331,18 @@ def _call_function(self, *args, **kwargs):
         if self.endpoint.first_arg_is_req:
             args = (request,) + args
 
+        needs_cleanup = False
```

Comment:
The variable name does not really help understanding what is going on.
What about `first_time` (being `not needs_cleanup`) or `retry` ?

Reply:
Rebased and renamed the variable to `first_time`.